backend-features/dev-mode.feature

```diff
@@ -6,7 +6,7 @@ Feature: Development mode
     Given I have no service configuration
     When I start the service with "--dev" without NATS
     Then the service should start
-    And the service should be configured with "NATS_URL" set to "localhost:4222"
+    And the service should be configured with "NATS_URL" set to either "localhost:4222" or "nats://localhost:4222"
```

Comment:
That's a good point.
We need to test that every backend can handle both the full url **and** only the hostname+port part correctly.
But I think that's the scope of another pull request.

Can you please revert that change and implement proper handling of NATS_URL?